cachi2/core/package_managers/pip.py

```diff
@@ -1533,8 +1444,8 @@ def _download_pypi_package(requirement, pip_deps_dir, pypi_proxy_url, pypi_proxy
 
     :param PipRequirement requirement: PyPI requirement from a requirement.txt file
     :param Path pip_deps_dir: The deps/pip directory in a Cachito request bundle
-    :param str pypi_proxy_url: URL of Nexus PyPI proxy
-    :param requests.auth.AuthBase pypi_proxy_auth: Authorization for the PyPI proxy
+    :param str pypi_proxy_url: URL of the PyPI server or a proxy
```

Comment:
`pypi_proxy_url` -> pypi_url

Reply:
Apart from that small detail, I didn't see any problems.
Great job Adam! 

RIP Nexus :coffin:

Reply:
Well spotted, fixed :+1: